model/qti/container/Container.php

```diff
@@ -150,6 +150,8 @@ public function getBody(){
      * modify the content of the body
      * 
      * @param string $body
+     * @param bool $integrityCheck
+     * @return bool
      */
     public function edit($body, $integrityCheck = false){
```

Comment:
Could we just add a call to the common logger to identify the content is being edited programmaticaly?